src/pages/NotFound.js

```diff
@@ -1,19 +1,21 @@
-import { NavLink } from 'react-router-dom';
+import { useNavigate } from 'react-router-dom';
 import { GoHome, GoChevronLeft } from 'react-icons/go';
 import styled, { keyframes } from 'styled-components';
+import Button from '../components/Button';
 
 export default function NotFound() {
+  const navigate = useNavigate();
   return (
     <Wrapper>
       <AnimatedGradientText>ERROR 404</AnimatedGradientText>
       <span>🎉🥳</span>
       <span>Herzlichen Glückwunsch!</span>
       <span>Du hast eine Seite gefunden, die es nicht gibt!</span>
```

Comment:
Really nice text :-)) - line 13 i would switch to `p`element

Reply:
i changed it to `p` too :)